sorc/ncep_post.fd/UPP_PHYSICS.f

```diff
@@ -2638,6 +2641,705 @@ SUBROUTINE CALGRADPS(PS,PSX,PSY)
+
+      real,dimension(0:14) :: tm, rhm
+
+      real,dimension(0:30), parameter        :: co1 = &
+       (/0.0, -.2926, .0070, -.0099, .0358, .0356, .0353, .0333, .0291, &
+         .0235, .0169, .0060, -.0009, -.0052, -.0079, -.0093,&
+        -.0116, -.0137, .0030, .0033, -.0005, -.0024, -.0023,&
+        -.0021, -.0007, .0013, .0023, .0024, .0012, .0002, -.0010/)
+
+      real,dimension(0:30), parameter        :: co2 = &
+       (/0.0, -9.7961, .0099, -.0222, -.0036, -.0012, .0010, .0018, .0018,&
+         .0011, -.0001, -.0016, -.0026, -.0021, -.0015, -.0010,&
+         -.0008, -.0017, .0238, .0213, .0253, .0232, .0183, .0127,&
+          .0041, -.0063, -.0088, -.0062, -.0029, .0002, .0019/)
+
+      real,dimension(0:30), parameter        :: co3 = &
+       (/0.0, 5.0037, -0.0097, -.0130, -.0170, -.0158, -.0141, -.0097,&
+        -.0034, .0032, .0104, .0200, .0248, .0273, .0280, .0276,&
+        .0285, .0308, -.0036, -.0042, -.0013, .0011, .0014, .0023,&
+        .0011, -.0004, -.0022, -.0030, -.0033, -.0031, -.0019/)         
+
+      real,dimension(0:30), parameter        :: co4 = &
+       (/0.0, -5.0141, .0172, -.0267, .0015, .0026, .0033, .0015, -.0007,&
+        -.0030, -.0063, -.0079, -.0074, -.0055, -.0035, -.0015,&
+        -.0038, -.0093, .0052, .0059, .0019, -.0022, -.0077, -.0102,&
+        -.0109, -.0077, .0014, .0160, .0217, .0219, .0190/)
+
+      real,dimension(0:30), parameter        :: co5 = &
+       (/0.0, -5.2807, -.0240, .0228, .0067, .0019, -.0010, -.0003, .0012,&
+         .0027, .0056, .0067, .0067, .0034, .0005, -.0026, -.0039,&
+         -.0033, -.0225, -.0152, -.0157, -.0094, .0049, .0138,&
+         .0269, .0388, .0334, .0147, .0018, -.0066, -.0112/)
+
+      real,dimension(0:30), parameter        :: co6 = &
+       (/0.0, -2.2663, .0983, .3666, .0100, .0062, .0020, -.0008, -.0036,&
+         -.0052, -.0074, -.0086, -.0072, -.0057, -.0040, -.0011,&
+         .0006, .0014, .0012, -.0005, -.0019, .0003, -.0007, -.0008,&
+         .0022, .0005, -.0016, -.0052, -.0024, .0008, .0037/)
+
+      type(all_grids), dimension(ista:iend,jsta:jend,0:lsm) :: tmpk_grids, rh_grids
+      integer,         dimension(ista:iend,jsta:jend,0:lsm) :: tmpk_levels, rh_levels
+
+      real,dimension(ista:iend,jsta:jend)    :: hprob,mprob,lprob
+      real,dimension(ista:iend,jsta:jend)    :: slrgrid, slrgrid2
+      real,dimension(ista:iend,jsta:jend)    :: psfc,pres,qpf,swnd,prp
+
+      character*20 nswFileName
+      real :: psurf,p,sgw,sg1,sg2,dtds,rhds
+      real :: f1,f2,f3,f4,f5,f6
+      real :: p1,p2,p3
+      real :: hprob_tot
+      real :: mprob_tot
+      real :: lprob_tot
+
+      integer :: i,j,k,ks,L,LL,imo,iday
+!
+!***************************************************************************
+!
+! day and month of the year
+
+      imo = idat(1)
+      iday= idat(2)
+
+! climatology
+! currently not used, snoden climatology files saved in fix directory
```

Comment:
@JesseMeng-NOAA May I know why you commented out following lines of  code instead of removed them?

Reply:
SLR climatology is part of the WPC product. I commented it out because we don't use it for now but maybe worth it to keep the information. Please let me know if you want it removed.

Reply:
@JesseMeng-NOAA Thanks for explaining. You may keep them. The same reason for these 2.5km_climo_snoden* files under fix/Breadboard/?

Reply:
> @JesseMeng-NOAA Thanks for explaining. You may keep them. The same reason for these 2.5km_climo_snoden* files under fix/Breadboard/?

YES